README.md

```diff
@@ -213,6 +213,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [IP.City](https://ip.city) — 100 Free IP geolocation requests per day
   * [Abstract API](https://www.abstractapi.com) — API suite for various use cases, including IP geolocation, gender detection, or email validation.
   * [Apify](https://www.apify.com/) — Web scraping and automation platform to create an API for any website and extract data. Ready-made scrapers, integrated proxies, and custom solutions. Free plan with $5 platform credits included every month.
+  * [Automatio](https://automatio.co) — Web Automation & Data Scraping Without Coding. It comes with **5 free credits**, equal to 1.3 hours of working instance/bot.
```

Comment:
no clear pricing on the site.

Reply:
I realized that myself that is requirement after I submitted it. We have ne page coming out soon, which includes that. Will come back here and update it after we release it.

Thanks

![image](https://github.com/user-attachments/assets/c6051465-c83e-4c96-8cc3-d8e387084b04)